Examples/antsApplyTransforms.cxx

```diff
@@ -849,6 +880,32 @@ static void antsApplyTransformsInitializeCommandLineOptions( itk::ants::CommandL
   parser->AddOption( option );
   }
 
+  {
+  std::string description =
+    std::string( "Output image data type. " )
+    + std::string( "This is a direct typecast; output values are not rescaled. " )
+    + std::string( "Default is to use the internal data type (float or double). " )
+    + std::string( "uchar is unsigned char; others are signed. " )
+    + std::string( "WARNING: Outputs will be incorrect (overflowed/reinterpreted) " )
+    + std::string( "if values exceed the range allowed by your choice. " )
+    + std::string( "Note that some pixel types are not supported by some image " )
+    + std::string( "formats. e.g. int is not supported by jpg. " );
+
+  OptionType::Pointer option = OptionType::New();
+  option->SetLongName( "output-data-type" );
+  option->SetShortName( 'u' );
+  option->SetUsageOption( 0, "Linear" );
```

Comment:
Somehow this snuck in?

Reply:
The short name? Yes, I mentioned that in the commit message. --output-data-type is very long. It seemed reasonable to choose a short name also, since everything else except for --float has one.

Reply:
`` option->SetUsageOption( 0, "Linear" );``

Reply:
Whoops. Yes, that was just a paste error when adding the short name. Removed, thanks.